share/pot/stop.sh

```diff
@@ -5,12 +5,16 @@
 stop-help()
 {
 	cat <<-"EOH"
-	pot stop [-hv] -p potname [pname]
+	pot stop [-hv] [-p] potname
```

Comment:
This change makes things worse, as you specify the syntax so that: `pot stop -pv potname` should be legal and work (which it won't). Also, the explicit definition of precedence is missing.

Maybe `pot stop [-hv] [-p potname] [potname]` would be more correct (I left "pname" earlier to allow referencing the different parameters).